packages/navbrand/src/react/index.tsx

```diff
@@ -82,9 +56,17 @@ const NavBrand = React.forwardRef<NavBrandElement, NavBrandProps>(
 export default NavBrand
 
 const Logo: React.FC = props => (
-  <div {...styles.logo(Boolean(props.children))} {...props} />
+  <div
+    className={classNames(
+      'psds-navbrand__logo',
+      Boolean(props.children) && 'psds-navbrand__logo--wordmark'
```

Comment:
Not sure it's any safer, but `React.Children.count(props.children) > 0` comes to mind.